src/main/java/org/jitsi/jigasi/CallManager.java

```diff
@@ -459,7 +518,15 @@ public synchronized static void hangupCall(Call   call,
         hangupCallThread.reasonCode = reasonCode;
         hangupCallThread.reason = reason;
 
-        threadPool.submit(hangupCallThread);
+        // if we are unhealthy, let's process error hangups
+        if (!healthy && reasonCode > 400)
```

Comment:
Why the `reasonCode > 400` check? The only case where `healthy==false`, we suspect `submit()` will fail with an exception.

Reply:
Ok, I will drop that check.